src/task.c

```diff
@@ -602,13 +615,16 @@ task_run (AppData *app_data)
         entry_point = g_strdup_printf ("%s %s", TASK_PLUGIN_SCRIPT, DEFAULT_ENTRY_POINT);
     }
     if (task->metadata->nolocalwatchdog) {
-      task->remaining_time = HEARTBEAT;
+      task->remaining_time = 0;
```

Comment:
So the solution is to disable the process timeout callback when the local watchdog is disabled, right?

Reply:
That's correct.